config/uplink.yaml

```diff
@@ -13,11 +13,6 @@ ingress:
     - secretName: uplink-tls
       hosts:
         - uplink.jenkins.io
-image:
-  # repo & '@sha256' suffix here as we're retrieving a docker digest from updatecli
-  repository: jenkinsciinfra/uplink@sha256
-  tag: c7ecad0e5e24bc05ac722418faf844d6a0286620c6316ff712459ecfcc3ee7ce
-  pullPolicy: IfNotPresent
```

Comment:
`image` section removed from the values to keep those defined in the chart default values.

Note: this Docker image tag wasn't tracked by updatecli, no manifest cleanup needed.